environment.yml

```diff
@@ -13,6 +13,7 @@ dependencies:
   - bzip2
   - cmake >=3.15
   - cspice
+  - csm==3.0.3
```

Comment:
For some reason this isn't resolving correctly https://astroservices.usgs.gov/jenkins/job/USGS-Astrogeology/job/ISIS3/job/PR-4130/1/execution/node/44/log/

Reply:
This is a feature branch, so we can merge as needed. I don't see a problem with merging for now. As soon as I figure out what is causing it, I'll put up another PR.

Reply:
I changed it to allow any 3.0.3.X version, and it now resolves the packages.